### ISSUE TITLE
it seems that results are only the psnr and ssim

### ISSUE THREAD

**Issue by robbiewongBD** [rwenqi/NBD-GLRA#3]
it seems that results are only the psnr and ssim, without showing the restored image
![image](https://user-images.githubusercontent.com/42163163/50344025-37a5ec00-0564-11e9-9fdc-444c3053158c.png)

**Comment by robbiewongBD** [comment]
![image](https://user-images.githubusercontent.com/42163163/50345801-447a0e00-056b-11e9-9d58-6b7ca4363bfc.png)
what to do with this error.

**Comment by rwenqi** [comment]
it has been updated.

**Comment by robbiewongBD** [comment]
> it has been updated.

is this error realated to the matcaffe configuration?
![image](https://user-images.githubusercontent.com/42163163/50390212-457b8d00-076f-11e9-8765-0ddbebb6fe99.png)

**Comment by rwenqi** [comment]
I used MATLAB 2014 and the caffe verson of https://github.com/Liusifei/caffe-lowlevel

**Comment by MaxChanger** [comment]
> is this error realated to the matcaffe configuration?

I also meet this problem, do you slove it ？

**Comment by robbiewongBD** [comment]
@MaxChanger 还没有，我微信ID：wangbodi764328784，可以加来一起讨论。

**Comment by rwenqi** [comment]
I‘ll check it, thanks for the question.

**Comment by MaxChanger** [comment]
I use the caffe version which you offered (caffe-lowlevel), compiled and passed, but I found a new problem.
![image](https://user-images.githubusercontent.com/35401968/50684774-afd6dc00-1051-11e9-978f-273a60f664f6.png)

Another question I want to ask is , if I search the " Solver.Solver_.net.get_weights() " in Google, I find much information is about your project and no one else used it. In other words, I can't find solutions in other ways, only through you and your team.

I hope this problem can be solved as soon as possible. Thanks！

**Comment by LiuzhuForFun** [comment]
> I use the caffe version which you offered (caffe-lowlevel), compiled and passed, but I found a new problem.
> ![image](https://user-images.githubusercontent.com/35401968/50684774-afd6dc00-1051-11e9-978f-273a60f664f6.png)
> 
> Another question I want to ask is , if I search the " Solver.Solver_.net.get_weights() " in Google, I find much information is about your project and no one else used it. In other words, I can't find solutions in other ways, only through you and your team.
> 
> I hope this problem can be solved as soon as possible. Thanks！

what a similar common

**Comment by LiuzhuForFun** [comment]
@MaxChanger ,Have you solve this problem？ What other state-of-the-art methods you know？Thanks!

**Comment by MaxChanger** [comment]
No, I am sorry that I did not continue to follow the relevant work.
Gook luck~

**Comment by LiuzhuForFun** [comment]
` w = net.layers('conv0').params(1);
    weights = w.get_data();
    for k = 1:50
        for j = 1:50
            weights(1,k,1,j) = M(k, j);
        end
    end
    w.set_data(weights);`
I use the standard caffe and I modified codes like this. @rwenqi @robbiewongBD  Is that correct?
